epubhv/cli.py

```diff
@@ -21,6 +21,35 @@ def main():
         help="change all the epub files to hortical",
     )
 
+    parser.add_argument(
+        "--convert",
+        dest="convert",
+        choices=[
+            "s2t", "t2s", 's2tw', 
+            'tw2s', 's2hk', 'hk2s', 
+            's2twp', 'tw2sp', 't2tw',
+            'hk2t', 't2hk', 't2jp','jp2t', 'tw2t'],
+        help='''
```

Comment:
add some basic help here?

s2t: Simplified Chinese to Traditional Chinese
t2s: Traditional Chinese to Simplified Chinese
s2tw: Simplified Chinese to Traditional Chinese (Taiwan Standard)
tw2s: Traditional Chinese (Taiwan Standard) to Simplified Chinese
s2hk: Simplified Chinese to Traditional Chinese (Hong Kong variant)
hk2s: Traditional Chinese (Hong Kong variant) to Simplified Chinese
s2twp: Simplified Chinese to Traditional Chinese (Taiwan variant)
tw2sp: Traditional Chinese (Taiwan variant) to Simplified Chinese
t2tw: Traditional Chinese (OpenCC Standard) to Taiwan Standard
hk2t: Traditional Chinese (Hong Kong variant) to Traditional Chinese
t2hk: Traditional Chinese (OpenCC Standard) to Hong Kong variant
t2jp: Traditional Chinese Characters (Kyūjitai) to New Japanese Kanji
jp2t: New Japanese Kanji to Traditional Chinese Characters (Kyūjitai)
tw2t: Traditional Chinese (OpenCC Standard) to Traditional Chinese (Taiwan standard)